src/tools/perf/perftest.c

```diff
@@ -733,7 +759,8 @@ static ucs_status_t setup_mpi_rte(struct perftest_context *ctx)
     rte_group_t group;
 
     rte_init(NULL, NULL, &group);
-    if (1 == rte_group_rank(group)) {
+    if (1 == rte_group_rank(group) ||
+             ctx->params.super.flags & UCX_PERF_TEST_FLAG_LOOPBACK) {
```

Comment:
Alignment (see `clang-format`).

Reply:
now that rte group size is 1 for loopback case, do we really need to check loopback flag?
can just check `if (rte_group_rank(group) == (rte_group_size(group) - 1))` ?